learning/katas/go/Core Transforms/Additional Outputs/Additional Outputs/task.md

```diff
@@ -16,12 +16,10 @@
   ~ limitations under the License.
   -->
 
-# Side Output
+# Additional Outputs
 
-While ParDo always produces a main output PCollection (as the return value from apply), you can 
```

Comment:
This paragraph is actually from the Beam Programming Guide.
https://beam.apache.org/documentation/programming-guide/#additional-outputs
Probably should consider revising the guide as well if it's not accurate, e.g. main output.

FYI, historically it used to be called Side Output.

Reply:
I'll revise the name in other Katas as well

Reply:
I remember. Side Output was a historical holdover from the original Dataflow API, and by the nature of how extra outputs are handled by the Java SDK. Thanks!